packages/service-library/tests/fastapi/test_rabbitmq.py

```diff
@@ -0,0 +1,147 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+
+import pytest
+from faker import Faker
+from models_library.rabbitmq_messages import LoggerRabbitMessage, RabbitMessageBase
+from tenacity.retry import retry_if_exception_type
+from tenacity.stop import stop_after_delay
+from tenacity.wait import wait_fixed
+
+_TENACITY_RETRY_PARAMS = {
+    "reraise": True,
+    "retry": retry_if_exception_type(AssertionError),
+    "stop": stop_after_delay(30),
+    "wait": wait_fixed(0.1),
+}
+
+# Selection of core and tool services started in this swarm fixture (integration)
+pytest_simcore_core_services_selection = [
+    "rabbit",
+]
+
+pytest_simcore_ops_services_selection = []
+
+
+@pytest.fixture
+def rabbit_log_message(faker: Faker) -> LoggerRabbitMessage:
+    return LoggerRabbitMessage(
+        user_id=faker.pyint(min_value=1),
+        project_id=faker.uuid4(),
+        node_id=faker.uuid4(),
+        messages=faker.pylist(allowed_types=(str,)),
+    )
+
+
+@pytest.fixture(params=["rabbit_log_message"])
+def rabbit_message(
+    request: pytest.FixtureRequest,
+    rabbit_log_message: LoggerRabbitMessage,
+) -> RabbitMessageBase:
+    return {
+        "rabbit_log_message": rabbit_log_message,
+    }[request.param]
+
+
+# # https://github.com/ITISFoundation/osparc-simcore/issues/5059
```

Comment:
OMG!! what is this :-D
Man, you cannot copy this code e.g. in the issue you put there? You prefer to add bytes in comments :-)))

Reply:
I don't know what this is. Somehow these test are here, but all the fixtures are missing. So I guess I have to implement those...